src/tlo/methods/hiv.py

```diff
@@ -597,6 +602,11 @@ def _build_linear_models(self):
             p["prob_circ_after_hiv_test"],
             Predictor("hv_inf").when(False, 1.0).otherwise(0.0),
             Predictor("sex").when("M", 1.0).otherwise(0.0),
+            Predictor("year",
+                      external=True,
+                      conditions_are_mutually_exclusive=True,
+                      conditions_are_exhaustive=True).when("<2019", 1)
```

Comment:
as we're using `otherwise`, I don't think the we can say that the (`when`) conditions are exhaustive
```suggestion
                      ).when("<2019", 1)
```